### PR TITLE
Add customer order display via notifications

### PR DESCRIPTION
**Pull request by Raghu150999**
- added notification for customer order
- simple order display activity
- added merchant server
- server fcm integration

### CODE REVIEW

File: merchant-app/app/src/main/java/com/googleinterns/smb/adapter/OrderAdapter.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    private List<Boolean> availableItems;
+    private PriceChangeListener mListener;
+
+    public OrderAdapter(PriceChangeListener listener, List<BillItem> billItems, FragmentManager fragmentManager) {
+        mFragmentManager = fragmentManager;
+        this.billItems = billItems;
+        availableItems = new ArrayList<>();
+        for (int i = 0; i < billItems.size(); i++) {
+            availableItems.add(true);
+        }
```
Comment (pulkit-singhal) [inline merchant-app/app/src/main/java/com/googleinterns/smb/adapter/OrderAdapter.java:46]: Use Collections.fill

File: merchant-app/app/src/main/java/com/googleinterns/smb/adapter/OrderAdapter.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    /**
+     * Compute total price by summing price for each item
+     */
+    private Double getTotalPrice() {
+        Double totalPrice = 0.0;
+        for (int i = 0; i < billItems.size(); i++) {
+            if (availableItems.get(i)) {
+                totalPrice += billItems.get(i).getTotalPrice();
+            }
+        }
```
Comment (pulkit-singhal) [inline merchant-app/app/src/main/java/com/googleinterns/smb/adapter/OrderAdapter.java:61]: This seems more to me like Stream (Filter, Map, Collect) Usecase

Comment (Raghu150999) [reply merchant-app/app/src/main/java/com/googleinterns/smb/adapter/OrderAdapter.java:61]: I tried using that, but android studio complains about requirement of android API level >= 24 for using that.

File: merchant-app/app/src/main/java/com/googleinterns/smb/adapter/OrderAdapter.java (hunk 1 above)
Comment (Raghu150999) [reply merchant-app/app/src/main/java/com/googleinterns/smb/adapter/OrderAdapter.java]: done

**Pull request merged**